client/src/Pos/Client/Txp/Util.hs

```diff
@@ -688,8 +684,10 @@ stabilizeTxFee pendingTx linearPolicy utxo outputs = do
     minFee <- fixedToFee (txSizeLinearMinValue linearPolicy)
     mtx <- stabilizeTxFeeDo (False, firstStageAttempts) minFee
     case mtx of
-        Nothing -> throwError FailedToStabilize
-        Just tx -> pure $ tx & \(S.Min (S.Arg _ txRaw)) -> txRaw
+        Nothing ->
+            throwError FailedToStabilize
+        Just (S.Min (S.Arg _ txRaw)) ->
+            pure txRaw
```

Comment:
I found the `tx & \(S.Min (S.Arg _ txRaw))` thing confusing so I made it more ordinary